core/src/test/java/org/elasticsearch/indices/analysis/AnalysisModuleTests.java

```diff
@@ -332,7 +346,8 @@ public void testUnderscoreInAnalyzerNameAlias() throws IOException {
                 .put("index.analysis.analyzer.valid_name.tokenizer", "keyword")
                 .put("index.analysis.analyzer.valid_name.alias", "_invalid_name")
                 .put(Environment.PATH_HOME_SETTING.getKey(), createTempDir().toString())
-                .put(IndexMetaData.SETTING_VERSION_CREATED, "1")
+                // analyzer aliases are only allowed for indices created in v5.0 alpha5 or before
+                .put(IndexMetaData.SETTING_VERSION_CREATED, VersionUtils.randomVersionBetween(random(), null, Version.V_5_0_0_alpha5))
```

Comment:
We don't explicitly support upgrading between alphas. Can you test this with a 2.x version instead? I would use `VersionUtils.getPreviousVersion(Version.V_5_0_0_alpha1)`